public/app/features/dataframe-import/utils.ts

```diff
@@ -1,55 +1,31 @@
-import { Accept } from 'react-dropzone';
 import { Observable } from 'rxjs';
 
-import { toDataFrame } from '@grafana/data';
+import { readCSV, toDataFrame } from '@grafana/data';
 
 import { FileImportResult } from './types';
 
-function getFileExtensions(acceptedFiles: Accept) {
-  const fileExtentions = new Set<string>();
-  Object.keys(acceptedFiles).forEach((v) => {
-    acceptedFiles[v].forEach((extension) => {
-      fileExtentions.add(extension);
-    });
-  });
-  return fileExtentions;
-}
-
-export function formatFileTypes(acceptedFiles: Accept) {
```

Comment:
Didn't seem to be used, except by the test https://github.com/search?q=repo%3Agrafana%2Fgrafana+formatFileTypes&type=code